pkg/storage/trb_request_form.go

```diff
@@ -137,29 +137,41 @@ func (s *Store) GetTRBRequestFormsByTRBRequestIDs(ctx context.Context, trbReques
 	return forms, nil
 }
 
-// GetFundingSourcesByRequestID queries the DB for all the TRB request form funding sources
+// GetTRBFundingSourcesByRequestID queries the DB for all the TRB request form funding sources
 // matching the given TRB request ID
-func (s *Store) GetFundingSourcesByRequestID(ctx context.Context, trbRequestID uuid.UUID) ([]*models.TRBFundingSource, error) {
+func (s *Store) GetTRBFundingSourcesByRequestID(ctx context.Context, trbRequestID uuid.UUID) ([]*models.TRBFundingSource, error) {
 	fundingSources := []*models.TRBFundingSource{}
-	stmt, err := s.db.PrepareNamed(`SELECT * FROM trb_request_funding_sources WHERE trb_request_id=:trb_request_id`)
+	err := namedSelect(ctx, s, &fundingSources, sqlqueries.TRBRequestFundingSources.GetByTRBReqID, args{
+		"trb_request_id": trbRequestID,
+	})
+
 	if err != nil {
 		appcontext.ZLogger(ctx).Error(
 			"Failed to fetch TRB request funding sources",
 			zap.Error(err),
 			zap.String("trbRequestID", trbRequestID.String()),
 		)
-		return nil, err
+		return nil, &apperrors.QueryError{
```

Comment:
are we aiming to get rid of all the `apperrors` stuff? i was under the impression it's a tech-debt goal

Reply:
@ClayBenson94 ?

Reply:
My bad -- didn't catch this 🤦 

I noted in Slack, but for posterity I will share my thoughts here as well!

I think I typically would prefer standard go errors rather than trying to (or having to...) wrap errors all the time. Perhaps we could take a look at customizing error types in some other way, but I'd be good to move in the direction _away_ from these error types!

Reply:
I wouldn't hold up a PR for it because they're not really harmful, but maybe we could rip them out of the codebase at some point!